packages/pieces/community/linear/src/lib/common/props.ts

```diff
@@ -135,13 +136,39 @@ export const props = {
           });
 
           for (const label of labels.nodes) {
-            options.push({ label: label.name, value: label.id });
+            labelMap.set(label.id, label.name);
           }
 
           hasNextPage = labels.pageInfo.hasNextPage;
           cursor = labels.pageInfo.endCursor;
         } while (hasNextPage);
 
+        // Fetch all workspace labels 
+        hasNextPage = false;
+        cursor = undefined;
+
+        do {
+          const labels = await client.listIssueLabels({
+            orderBy: LinearDocument.PaginationOrderBy.UpdatedAt,
+            first: 100,
+            after: cursor,
+          });
+
+          for (const label of labels.nodes) {
+            // Prevent duplicates
+            if (!labelMap.has(label.id)) {
+              labelMap.set(label.id, label.name);
+            }
+          }
```

Comment:
can the id be duplicated? That's weird 🤔 
They don't use UUIDs? I would expect a workspace label ID to never collide with a team label ID